packages/common/src/utils/wallet.ts

```diff
@@ -134,3 +133,26 @@ export const convertWeiToWAudio = (amount: BN) => {
   const decimals = WEI_DECIMALS - SPL_DECIMALS
   return amount.div(new BN('1'.padEnd(decimals + 1, '0')))
 }
+
+/** USDC Utils */
+const BN_USDC_WEI = new BN('1000000')
+
+/** Formats a USDC wei string (full precision) to a fixed string suitable for
+display as a dollar amount. Note: WILL lose precision by rounding to nearest cent */
+export const formatUSDCWeiToUSDString = (amount: StringUSDC, precision = 2) => {
+  // Since we only need two digits of precision, we will multiply up by 100
+  // with BN, divide by $1 Wei, and then convert to JS number and divide back down
+  // before formatting to two decimal places.
+  const converted =
+    new BN(amount).muln(100).divRound(BN_USDC_WEI).toNumber() / 100
```

Comment:
Yeah I figured we would discuss this more holistically with Product at the next sync up. Realistically, we won't allow setting sub-cent values through the UI, so this would never matter. But you could always write an update to chain from elsewhere. And we would just need to be clear about the policy in those cases.

FWIW, I think ceiling is probably the best approach, as it allows us to follow a policy where you pay _exactly_ the amount that the track costs down to the last Wei. That would make access-checking easy, since we don't have to ensure that rounding logic exists and matches in all the places.